run.sh

```diff
@@ -3,51 +3,57 @@
 set -e
 set -o pipefail
 
-if [ "${AWS_ACCESS_KEY_ID}" = "**None**" ]; then
-  echo "You need to set the AWS_ACCESS_KEY_ID environment variable."
-  exit 1
-fi
-
-if [ "${AWS_SECRET_ACCESS_KEY}" = "**None**" ]; then
-  echo "You need to set the AWS_SECRET_ACCESS_KEY environment variable."
-  exit 1
-fi
-
-if [ "${AWS_BUCKET}" = "**None**" ]; then
-  echo "You need to set the AWS_BUCKET environment variable."
-  exit 1
-fi
-
-if [ "${PREFIX}" = "**None**" ]; then
-  echo "You need to set the PREFIX environment variable."
-  exit 1
-fi
-
-if [ -z "${MYSQL_ENV_MYSQL_USER}" ]; then
-  echo "You need to set the MYSQL_ENV_MYSQL_USER environment variable."
-  exit 1
-fi
-
-if [ -z "${MYSQL_ENV_MYSQL_PASSWORD}" ]; then
-  echo "You need to set the MYSQL_ENV_MYSQL_PASSWORD environment variable."
-  exit 1
-fi
-
-if [ -z "${MYSQL_PORT_3306_TCP_ADDR}" ]; then
-  echo "You need to set the MYSQL_PORT_3306_TCP_ADDR environment variable or link to a container named MYSQL."
-  exit 1
-fi
-
-if [ -z "${MYSQL_PORT_3306_TCP_PORT}" ]; then
-  echo "You need to set the MYSQL_PORT_3306_TCP_PORT environment variable or link to a container named MYSQL."
-  exit 1
-fi
+fail() {
+  # shellcheck disable=SC2068
+  echo -e $@
+}
+
+# use UTC for timestamps
+datecmd() {
+  # shellcheck disable=SC2068
+  date --utc $@
+}
+
+[[ -z "${AWS_ACCESS_KEY_ID+x}" ]] &&
+  fail "You need to set the AWS_ACCESS_KEY_ID environment variable."
+
+[[ -z "${AWS_SECRET_ACCESS_KEY+x}" ]] &&
```

Comment:
I thought about putting the var name + error message into a hash but I thought that might lesson that chance of this being accepted back upstream.